2024-10-09 16:29:00,528 Evaluation process is started.
python "C:\Users\DS1DPC2003M\Desktop\tool\neural_network_console3.10\libs\Python\Lib\site-packages\nnabla\utils\cli\cli.py" forward
	-c "C:\Users\DS1DPC2003M\Desktop\spresense_cv\data_not_square\raw_data_base.files\20241009_162840\results.nnp"
	-d "C:\Users\DS1DPC2003M\Desktop\spresense_cv\dataset4_not_square\test.csv"
	-o "C:\Users\DS1DPC2003M\Desktop\spresense_cv\data_not_square\raw_data_base.files\20241009_162840"
	--replace_path
2024-10-09 16:29:01,629 [nnabla]: [CALLBACK]: Exec forward on local
2024-10-09 16:29:01,646 [nnabla]: Using context "Context(backend=['cpu:float'], array_class='CpuCachedArray', device_id='')"
2024-10-09 16:29:01,689 [nnabla]: data 32 / 50
2024-10-09 16:29:01,699 [nnabla]: data 50 / 50
2024-10-09 16:29:01,703 [nnabla]: Forward Completed.
NNabla command line interface (Version:1.39.0.dev1, Build:231224031435, Callback:NNabla SSH callback module.)
